UnityProject/Assets/Scripts/Health/Living/DNABloodTypeGen.cs

```diff
@@ -0,0 +1,65 @@
+using Random = UnityEngine.Random;
+
+[System.Serializable]
+public class DNAandBloodType
+{
+	public string DNAString;
+	
+	public BloodTypes BloodType;
+	
+	private float BloodTypeGenerator;
+	
+	public DNAandBloodType()
+	{	
+		// Assigns DNA a GUID
+		DNAString = System.Guid.NewGuid().ToString();
+		// Rolls from 1 to 1000 to determine blood chance.
+		BloodTypeGenerator = Random.Range(1.0f,1000.0f);
```

Comment:
You can't use the unity random gen in a constructor. You may be able to use the rgen from System instead. An alternative way would be to do the rgen outside of the class and pass it to the constructor as a float param:

![image](https://user-images.githubusercontent.com/20813925/49703586-75487200-fc52-11e8-8270-ad511fe960fb.png)